generated/niscope/niscope_service.h

```diff
@@ -125,6 +125,12 @@ class NiScopeService final : public NiScope::Service {
   void Copy(const std::vector<NIComplexNumber_struct>& input, google::protobuf::RepeatedPtrField<niscope_grpc::NIComplexNumber>* output);
   void Copy(const NIComplexI16_struct& input, niscope_grpc::NIComplexInt32* output);
   void Copy(const std::vector<NIComplexI16_struct>& input, google::protobuf::RepeatedPtrField<niscope_grpc::NIComplexInt32>* output);
+  std::map<std::int32_t, std::int64_t> niscopeint64attributevalues_input_map_ { {0, 0},{1, 0},{2, 1}, };
+  std::map<std::int64_t, std::int32_t> niscopeint64attributevalues_output_map_ { {0, 0},{0, 1},{1, 2}, };
+  std::map<std::int32_t, double> niscopereal64attributevalues_input_map_ { {0, 0},{1, 0},{2, 1}, };
+  std::map<double, std::int32_t> niscopereal64attributevalues_output_map_ { {0, 0},{0, 1},{1, 2}, };
```

Comment:
In subsequent draft PR I will be trying out mapping using different mechanism which should solve confusing mapping of 0 to 1 and 1 to 2. If that doesn't work out, I will add something to prevent this enum from generating. Keeping the comment open till then.

Reply:
Like I said in the other PR, it would be good to find a way to not map 1->0 and 2->1.

Reply:
This weird mapping of 1 to 0 and 2 to 1 is now resolved, since we use unmapped values everywhere.